instrumentation/internal/internal-url-class-loader/javaagent-integration-tests/src/test/java/AddUrlTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+import java.lang.reflect.InvocationTargetException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.jupiter.api.Test;
+
+class AddUrlTest {
+
+  @Test
+  void testShouldInstrumentClassAfterItIsLoadedViaAddUrl()
```

Comment:
I would have used just `throws Exception` but I guess this is a matter of taste

Reply:
that is much cleaner, thanks for the suggestion